packages/rke2-cilium/generated-changes/patch/values.yaml.patch

```diff
@@ -200,7 +200,7 @@
 +portmapPlugin:
 +  image:
 +    repository: "rancher/hardened-cni-plugins"
-+    tag: "v1.5.1-build20240910"
++    tag: "v1.5.1-build20240830"
```

Comment:
cni-plugins were bumped here: https://github.com/rancher/rke2-charts/pull/521

But we missed the update to the [cilium-values.yaml.patch.template](https://github.com/rancher/rke2-charts/blob/af7c7c9a24c746866ba60717bc20ca21cf406b21/updatecli/scripts/cilium-values.yaml.patch.template#L203) file

Reply:
https://github.com/rancher/rke2-charts/pull/530